be/src/Project/Web/Seller/Seller.Web/Areas/Orders/Repositories/OrderAttributes/OrderAttributesRepository.cs

```diff
@@ -0,0 +1,234 @@
+﻿using Foundation.ApiExtensions.Services.ApiClientServices;
+using Foundation.GenericRepository.Paginations;
+using Foundation.Localization;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Options;
+using Seller.Web.Shared.Configurations;
+using Seller.Web.Areas.Orders.DomainModels;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Foundation.ApiExtensions.Communications;
+using Foundation.ApiExtensions.Models.Request;
+using Foundation.ApiExtensions.Models.Response;
+using Foundation.ApiExtensions.Shared.Definitions;
+using Foundation.Extensions.ExtensionMethods;
+using Foundation.Extensions.Exceptions;
+using Seller.Web.Areas.Orders.ApiRequestModels;
+using System.Linq;
+
+namespace Seller.Web.Areas.Orders.Repositories.OrderAttributes
+{
+    public class OrderAttributesRepository : IOrderAttributesRepository
+    {
+        private readonly IApiClientService _apiClientService;
+        private readonly IOptions<AppSettings> _options;
+        private readonly IStringLocalizer<GlobalResources> _globalLocalizer;
+
+        public OrderAttributesRepository(
+            IApiClientService apiClientService, 
+            IOptions<AppSettings> options, 
+            IStringLocalizer<GlobalResources> globalLocalizer)
+        {
+            _apiClientService = apiClientService;
+            _options = options;
+            _globalLocalizer = globalLocalizer;
+        }
+
+        public async Task DeleteAsync(string token, string language, Guid? id)
+        {
+            var apiRequest = new ApiRequest<RequestModelBase>
+            {
+                Language = language,
+                Data = new RequestModelBase(),
+                AccessToken = token,
+                EndpointAddress = $"{_options.Value.OrderUrl}{ApiConstants.Order.OrderAttributesApiEndpoint}/{id}"
+            };
+
+            var response = await _apiClientService.DeleteAsync<ApiRequest<RequestModelBase>, RequestModelBase, BaseResponseModel>(apiRequest);
+
+            if (!response.IsSuccessStatusCode && response?.Data != null)
```

Comment:
**Suggestion** (non-blocking): Please use "is false" and "is not" instead of ! and !=